docs/self-managed/modeler/web-modeler/configuration/configuration.md

```diff
@@ -181,10 +184,13 @@ The `webapp` component sends certain events (e.g. "user opened diagram", "user l
 
 ### Logging
 
-| Environment variable | Description                            | Example value                |
-| -------------------- | -------------------------------------- | ---------------------------- |
-| `LOG_FILE_PATH`      | [optional]<br/>Path to log file output | `/full/path/to/log/file.log` |
+| Environment variable | Description                                  | Example value                |
+| -------------------- | -------------------------------------------- | ---------------------------- |
+| `LOG_FILE_PATH`      | [optional]<br/>Path to log file output       | `/full/path/to/log/file.log` |
+| `LOG_LEVEL_CLIENT`   | [optional]<br/>Log level for the client      | `DEBUG`                      |
+| `LOG_LEVEL_WEBAPP`   | [optional]<br/>Log level for the node server | `DEBUG`                      |
```

Comment:
`LOG_LEVEL_CLIENT` and `LOG_LEVEL_SERVER` would have been more consistent maybe 🙃. But I'm not sure if it's worth renaming the variable in Web Modeler.

Reply:
Agree.

But I also do not think it's worth the effort to make the change in the code, test it and backport it (which is all done  currently). 🙃